sklearn/metrics/tests/test_common.py

```diff
@@ -638,7 +638,8 @@ def test_inf_nan_input():
     for metric in CLASSIFICATION_METRICS.values():
         for y_true, y_score in invalids:
             assert_raise_message(ValueError,
-                                 "Can't handle mix of binary and continuous",
```

Comment:
Ahh, sorry. I think I misinterpreted you. I thought you meant the error string could either be either-or. I'll be modifying the test cases soon. Thanks.